### PR TITLE
Switch from localstack to minIO

### LINKED ISSUE #2634: Switch to minio from localstack in local dev
Locastack is a very heavy tool and unreliable at times and most features are blocked behind a paywall, switch to using [minIO](https://github.com/minio/minio)

### PR DESCRIPTION
**Pull request by amjithtitus09**
## Proposed Changes

- fixes https://github.com/ohcnetwork/care/issues/2634
- Replaced docker localstack configuration with minIO configuration. Added script to creates facility and patient buckets. Converts facility bucket to public to allow public access as minIO doesn't support ACL.

### Associated Issue

- Switches from localstack to minIO for local dev s3 since localstack is a very heavy and unreliable at times and most features are blocked behind a paywall

## Merge Checklist

- [ ] Tests added/fixed
- [ ] Update docs in `/docs`
- [ ] Linting Complete
- [ ] Any other necessary step

_*Only PR's with test cases included and passing lint and test pipelines will be reviewed*_

@ohcnetwork/care-backend-maintainers @ohcnetwork/care-backend-admins

## Summary by CodeRabbit

- **New Features**
	- Introduced a new MinIO service for object storage, replacing the previous LocalStack service.
	- Added initialization and entrypoint scripts for setting up MinIO.

- **Bug Fixes**
	- Updated environment variables for bucket credentials and endpoints to ensure compatibility with the new MinIO service.

- **Documentation**
	- Updated setup documentation to reflect the transition from LocalStack to MinIO in the development environment.

- **Chores**
	- Enhanced end-of-line handling for shell scripts to ensure consistency across environments.

### CODE REVIEW

**Comment by amjithtitus09** [comment]
@vigneshhari Please review. 

Would like to add that minIO doesn't seem to support "public-read" ACL set during upload, hence 'facility-bucket' is initialized as public.

**Review by sainak — APPROVED** [review]

File: .gitattributes [hunk 1]
```diff
@@ -1,2 +1,3 @@
 * text=auto
 scripts/* text eol=lf
+docker/minio/* text eol=lf
```
Comment (rithviknishad) [inline .gitattributes]: Maybe at this point, it'd be easier to just declare all `*.sh` files with LF line endings

Comment (sainak) [reply .gitattributes]: yeah

Comment (sainak) [inline .gitattributes]:
```suggestion
*.sh text eol=lf 
```

**Review by rithviknishad — APPROVED** [review]

**Review by vigneshhari — APPROVED** [review]

**Pull request merged**